docs/docs/how-to/cloud/enable-image-cdn.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Enable Image CDN"
+description: "Learn how to accelerate builds locally and in Gatsby Cloud by offloading image processing to Gatsby's Image CDN"
+---
+
+Before continuing, make sure you understand [What is Image CDN](../../../reference/cloud/image-cdn/). Also, note that Image CDN only works on sites hosted on Gatsby cloud and doesn't work with other [hosting integrations](https://support.gatsbyjs.com/hc/en-us/articles/1500000907821-Hosting-and-Data-Source-Integrations).
```

Comment:
You can't link to the document like this, you have to use relative links like this:

```
/docs/reference/cloud/image-cdn
```